devstudio/installer/category.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<site>
+	<description>
+		JBoss Developer Studio Integration Stack Installer
+	</description>
+
+	<!-- copy IU's from JBDS Core Installer -->
+	<feature id="org.jboss.tools.bpel.feature" />
+	<feature id="org.testng.eclipse" />
+	<bundle id="com.jboss.devstudio.core.package" />
```

Comment:
This PR is just an option how to deal with:
1. Including additional dependencies into installer during the build rather than manually (see previous PR);
2. Including only what is actually required during installation to keep size of installer smaller.

Please, just reject this PR if it brings in annoying problems.